daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md

```diff
@@ -55,6 +55,14 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | prefetchCount  | N        | Number of messages to [prefetch](https://www.rabbitmq.com/consumer-prefetch.html). Consider changing this to a non-zero value for production environments. Defaults to `"0"`, which means that all available messages will be pre-fetched. | `"2"`
 | reconnectWait  | N        | How long to wait (in seconds) before reconnecting if a connection failure occurs | `"0"`
 | concurrencyMode | N        | `parallel` is the default, and allows processing multiple messages in parallel (limited by the `app-max-concurrency` annotation, if configured). Set to `single` to disable parallel processing. In most situations there's no reason to change this. | `parallel`, `single`
+| backOffPolicy              | N        | Retry policy, `"constant"` is a backoff policy that always returns the same backoff delay. `"exponential"` is a backoff policy that increases the backoff period for each retry attempt using a randomization function that grows exponentially. Defaults to `"constant"`. | `constant`、`exponential` |
+| backOffDuration            | N        | The fixed interval only takes effect when the policy is constant. | `"5s"`、`"5000"`                  |
+| backOffInitialInterval     | N        | The backoff initial interval on retry. Only takes effect when the policy is exponential. Defaults to `"500"`                         | `"50"`                       |
```

Comment:
Provide units in row. For example - "Time is in seconds"

Reply:
For all time type backoff parameters, they have a consistent form of parameter analysis. So I added a more detailed type description in the first appeared parameter `backOffDuration`. Do I need to bring this content with each parameter.

Reply:
Should be placed on every single one